with-vue-supabase/stepzen/postgresql/index.graphql

```diff
@@ -0,0 +1,47 @@
+type Demo {
+  created_at: DateTime
+  id: ID!
+  name: String
+}
+
+"""
+These are some examples of queries generated from the schema. Feel free to modify them or add your own.
+"""
+type Query {
+  getDemoList: [Demo]
```

Comment:
Could you include the `.sql` dump of the PostgreSQL database? That way other developers are able to clone this project, now they cannot..